users/albertia/index.md

```diff
@@ -8,29 +8,27 @@ author: Tony Alberti
 
 <img src="{{ site.url }}users/albertia/images/NUC16.jpg" width="450">
 
-I am a 3rd year graduate student here at OSU in Corvallis, OR working under the guidance of Todd Palmer. I began graduate school in January of 2014 and have been a full time student since. I defended my Masters of Science degree in October of 2015 and published my Masters thesis (see publications below) shortly thereafter.
+I am a 4th year graduate student here at OSU in Corvallis, OR working under the guidance of Todd Palmer. I began graduate school in January of 2014 and have since been a full time student collaborating on research with [Idaho National Laboratory](https://www.inl.gov/) (INL). I interned at INL in the summer of 2015 and 2016. There I worked with the [Rattlesnake/MAMMOTH Reactor Physics and Analysis group](https://moose.inl.gov/mammoth/SitePages/Home.aspx), specifically under the guidance of Javier Ortensi. I defended my Masters of Science degree in October of 2015 in nuclear reactor analysis and published my thesis (see publications below) shortly thereafter.
```

Comment:
Personally, I think the conversational tone here is good for the opener, but I would write the remaining three paragraphs more objectively, a little more businesslike. Less of "I", less conversational. For example: 

> Unfortunately, as one can imagine
> Rather than get into the details here

types of things are out of place. Great descriptions otherwise!

Reply:
I see your point, I feel as though Doug does the same thing. But I kind of like the conversational tone. I think that I want to keep my webpage sort of informal (but still get the point across), to seem more approachable to average joe blow who might read it. I leave the professionalism for the paper that I link.